superset/views/base.py

```diff
@@ -346,6 +346,16 @@ def common_bootstrap_payload() -> Dict[str, Any]:
 
     # should not expose API TOKEN to frontend
     frontend_config = {k: conf.get(k) for k in FRONTEND_CONF_KEYS}
+    frontend_config["EXCEL_EXTENSIONS"] = bool(
+        bool(conf["EXCEL_EXTENSIONS"].intersection(conf["ALLOWED_EXTENSIONS"])),
+    )
+    frontend_config["CSV_EXTENSIONS"] = bool(
+        bool(conf["CSV_EXTENSIONS"].intersection(conf["ALLOWED_EXTENSIONS"])),
+    )
+    frontend_config["COLUMNAR_EXTENSIONS"] = bool(
+        bool(conf["COLUMNAR_EXTENSIONS"].intersection(conf["ALLOWED_EXTENSIONS"])),
```

Comment:
This looks good, but to further simplify, wdyt about adding these keys to the FRONTEND_CONF_KEYS and then do the logic on the client side component? In other words, instead of this file being concerned with logic or the usage of this data, it just passes the data to the client and then the client only has to be concerned about the logic.

Reply:
sure to dry that out a bit it makes sense.

Reply:
However I'm going to mark this as a follow up as this will add some testing layer on frontend I think.

Reply:
ok, sounds good. thanks